pytamaro/it/operations.py

```diff
@@ -45,39 +53,39 @@ def componi(grafica_primopiano: Grafica, grafica_secondopiano: Grafica) \
     return compose(grafica_primopiano, grafica_secondopiano)
 
 
-def fissa(posizione_orizzontale: str, posizione_verticale: str,
-          grafica: Grafica) -> Grafica:
+def fissa(posizione_di_fissaggio: Punto, grafica: Grafica) -> Grafica:
     """
     Cambia la posizione di fissaggio di una grafica, ritornando una nuova
-    grafica con lo stesso contenuto ma una posizione di fissaggio aggiornata.
-
-    La nuova posizione di fissaggio è determinata dai parametri
-    `posizione_orizzontale` e `posizione_orizzontale`.
-
-    :param posizione_orizzontale: uno tra "sinistra", "centro", "destra" per
-           muovere la nuova posizione di fissaggio rispettivamente al bordo
-           sinistro, al centro (orizzontalmente), o al bordo destro della
-           grafica
-    :param posizione_verticale: uno tra "alto", "centro", "basso" per
-           muovere la nuova posizione di fissaggio rispettivamente al bordo
-           superiore, al centro (verticalmente), o al bordo inferiore della
-           grafica
+    grafica coon lo stesso contenuto ma una posizione di fissaggio aggiornata.
+    La nuova posizione di fissaggio è determinata dal parametro `posizione_di_fissaggio`
+
+    :param posizione_di_fissaggio: un oggetto di tipo punto che
+    identifica uno dei 9 punti d'interesse:
+    I punti di fissaggio validi sono:
```

Comment:
Do you think that something like this would work?
"Ogni grafica è racchiusa in un rettangolo di delimitazione, i punti di fissaggio validi corrispondono a: i quattro angoli, i punti centrali dei quattro lati e al centro di tale rettangolo"
And then I can list the exported names, it should be pretty easy to link the described points to the exported names.

Reply:
That sounds better!